### ISSUE TITLE
Openhab compatability

### ISSUE THREAD

**Issue by Bassysta** [dotvav/aasivak#7]
Hi.
Any possibility to make it work with Openhab?

**Comment by dotvav** [comment]
Hi @Bassysta, sorry I haven't paid attention to this repo in a few months. I have no experience with OpenHab, but I assume it can integrate with MQTT. You will need to bind the Aasivak/HiKumo topics to the right entity/values in OpenHab. If you do that, I suggest that you turn `mqtt_discovery` off in your local config.

**Comment by dotvav** [comment]
In other words, what work do you think is required to make this compatible with OpenHab?

**Comment by dotvav** [comment]
@Bassysta if you manage to integrate it with your openhab installation, would you mind sharing how you did it so that I can see if I can update the readme?

**Comment by Bassysta** [comment]
Hi Dotvav, sorry for my delay.
I'm not a Developer so i don't know to integrate it in Openhab correctly.
What i can say is that Openhab had a possibility to configure a broker that communicate with a local Mosquitto Server and that i can bind any mqtt topic to specific items(Button, DImmer, ecc), so maybe there is a possibility to configure your program "Out-of-thebox" of openhab and communicate with it using specific mqtt topic?

Have a nice day.

**Comment by Bassysta** [comment]
Hi.
I've readed a little bit more your README.
I've installed, started your program, that is configured now with my mqtt broker. But where i can find some logs to check any configuration problems? and a list of the mqtt topic/commands?

Of course after i've make it work with openhab i'll update you for the Readme.

Have a nice day

**Comment by Bassysta** [comment]
Last question, how can i create my own "api_user_agent" for api calls?

**Comment by Bassysta** [comment]
Hi dotvav. 
I just wanna said that your program works very well even with Openhab. No changes need to be done to your program.
Maybe next days i'll find the time to write down some info for the readme. for now, let me thanks for your work, i was searching for for a feature like this for a while.

Have a nice day.

**Comment by dotvav** [comment]
@Bassysta thank you for your comments. I haven't tried to document which topics are used because their name is dynamic (they depend on some internal Hi-Kumo identifiers) and I changed the structure slightly several times in the past. It doesn't matter to HomeAssistant because I updated the auto-discovery configuration at the same time. If these topics are exposed in the documentation, it means that I have to keep said documentation in sync as well. It shouldn't be too much work, though, and makes a lot of sense for users like yourself who are not using HomeAssistant auto-discovery.
